src/screens/menu.py

```diff
@@ -0,0 +1,122 @@
+import pygame
+
+from ._screen import Screen
+from src.types import State
+from ..config import SCREEN_WIDTH
+
+class MenuScreen(Screen):
+    def __init__(self, game) -> None:
+        super().__init__(game)
+
+        self.ship_count = None
+
+        button_text_color = (255, 255, 255)
+        button_bg_color = (0, 128, 255)
+        button_hover_color = (0, 200, 255)
```

Comment:
(note)
Just for consistency, we can probably put these in the Color class with the other colors. Or maybe give these a descriptive name to what colors these represent?